testssl.sh

```diff
@@ -760,8 +760,17 @@ fileout() {
      if ( "$do_pretty_json" && [[ "$1" == "service" ]] ) || show_finding "$severity"; then
          local finding=$(strip_lf "$(newline_to_spaces "$(strip_quote "$3")")")
          [[ -f "$JSONFILE" ]] && (fileout_json_finding "$1" "$severity" "$finding" "$cve" "$cwe" "$hint")
-         "$do_csv" && \
-              echo -e \""$1\"",\"$NODE/$NODEIP\",\"$PORT"\",\""$severity"\",\""$finding"\",\""$cve"\",\""$cwe"\",\""$hint"\"" >> "$CSVFILE"
+
+         if "$do_csv"; then
+            if "$do_test_just_one"; then
```

Comment:
yes, we could change this to ~ ``do_cipher_matched``. ``test_just_one()``. That anyway wasn't consistent and then would be needed to be renamed then to ``run_cipher_matched()``.